test/util/schemaform-utils.jsx

```diff
@@ -15,6 +15,9 @@ export class DefinitionTester extends React.Component {
     const uiSchema = props.uiSchema;
     const data = props.data;
     let schema = updateRequiredFields(props.schema, uiSchema, data, state);
+    if (props.definitions) {
```

Comment:
It's also possible that a specific page would have definitions already in its schema, so we probably want to merge the two like in the reducer code:

`schema = _.assign({ definitions: props.definitions }, schema);`